target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -14,6 +14,7 @@ case "$FIRMWARE" in
 	araknis,an-500-ap-i-ac|\
 	araknis,an-700-ap-i-ac|\
 	engenius,eap1200h|\
+	engenius,eap1750h|\
 	engenius,enstationac-v1|\
```

Comment:
It's better to convert calibration data to nvmem-cells. https://github.com/openwrt/openwrt/commit/48625a04453b4f5775db2d2a5abebf977b81344c

Reply:
thanks for reminding me about that new method, it works